tools/local-presubmit

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# A basic script to locally semi-simulate the build and test
+# operations performed by our continuous integration service. Note
+# that this does not do a clean and reinstall of npm modules and
+# sigh/bazel artifacts.
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
+    echo -e "$MAG$1$END"
+}
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+status "Testing all of the things."
+
+./tools/sigh && \
```

Comment:
https://github.com/PolymerLabs/arcs/blob/0ec76ed0c9d0d50bf0ab6250f096f225b1a8db37/tools/sigh.ts#L87

Sigh's default does a lot of the subsequent steps. Maybe this could be `tools/sigh webpack`?

Reply:
Done.